common/phal_L4/spi/spi.c

```diff
@@ -135,9 +169,47 @@ void DMA1_Channel3_IRQHandler()
     }
 }
 
+void DMA2_Channel2_IRQHandler()
+{
+    if (DMA2->ISR & DMA_ISR_TEIF3)
+    {
+        DMA2->IFCR |= DMA_IFCR_CTEIF3;
```

Comment:
Just glancing at this, thinking all these threes should be 2’s for channel 2, please look into this